transport_nantes/topicblog/models.py

```diff
@@ -1678,3 +1680,30 @@ def get_edit_url(self):
                 "topicblog:edit_panel",
                 kwargs={"pkid": self.pk, "the_slug": self.slug},
             )
+
+
+def get_random_panel():
```

Comment:
This would fit quite well in the Class directly, in my opinion

Reply:
If it were in the class, I'd expect it to manipulate an individual member object.  But instead it's doing a query.  It wouldn't every make sense to call

```
my_panel.get_random_panel()
```

Reply:
That's a pattern I've seen, but whatever sits right to you, it doesn't really matter :)